src/extension/wps/wps-core/src/main/java/org/geoserver/wps/executor/RemoteRequestInputProvider.java

```diff
@@ -60,7 +61,8 @@ public RemoteRequestInputProvider(
     protected Object getValueInternal(ProgressListener listener) throws Exception {
         InputReferenceType ref = input.getReference();
         URL destination = new URL(ref.getHref());
-
+        if(destination.getProtocol().toLowerCase().startsWith("http"))
```

Comment:
Hum what about FTP? Is it even supported?

Reply:
dont think so

Reply:
in the 05-007r7_Web_Processing_Service_WPS_v1.0.0.pdf page 31, there is activity diagram with FTP 
So I think the FTP should be covered.
I don't understand why there is the if statement, I would test all destinations of all protocols. I think all has to be whitelisted including file, http, ftp or whatever new there will be available otherwise it is security hole.
@aaime  Is there something I don't see?